wander-connect-sdk/src/wander-connect.ts

```diff
@@ -544,6 +557,64 @@ export class WanderConnect {
     this._close();
   }
 
+  public signOut(): Promise<void> {
+    if (this.authInfo.authStatus === "not-authenticated" || this.authInfo.authStatus === "loading") {
+      throw new Error("The user is not authenticated");
+    }
+
+    return new Promise((resolve, reject) => {
+      const { iframeRef } = this;
+      const contentWindow = iframeRef?.contentWindow;
+
+      if (!iframeRef || !contentWindow) {
+        reject(new Error("Missing Wander Connect iframe"));
+        return;
+      }
+
+      isomorphicSendMessage({
+        destination: "background",
+        // @ts-ignore
+        messageId: "embedded_signOut",
+        // @ts-ignore
+        data: undefined,
+      });
+
+      this.signOutMethodCallback = resolve;
+    });
+  }
+
+  public setTheme(theme: ThemeSetting): void {
```

Comment:
I think this function should check if the passed theme is light, dark or system as I passed random text and I could see it being set in localStorage.

Reply:
Added validation for the values.